girder/api/v1/resource.py

```diff
@@ -190,15 +190,21 @@ def _lookUpPath(self, path, user):
             parent = self.model('user').findOne({'login': username})
 
             if parent is None:
-                raise RestException('User not found: %s' % username)
+                if test is False:
```

Comment:
I think these blocks should be rewritten to be more Pythonic, like

    if test:
        return None
    else:
        raise RestException('User not found: %s' % username)